python/Ganga/Lib/Localhost/LocalHostExec.py

```diff
@@ -88,14 +88,18 @@
 
 import time #datetime #disabled for python2.2 compatiblity
 
+if os.path.isfile(os.path.abspath(appscriptpath[0])):
```

Comment:
No, I ended up adding this to the Dirac backend in order to get anything done sensibly during testing.
I can merge #418 first which is this short change but I was planning to mention that this supersedes it.